clients/filesystem-hadoop/src/main/java/com/datastrato/gravitino/filesystem/hadoop/Gvfs.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.filesystem.hadoop;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.DelegateToFileSystem;
+
+public class Gvfs extends DelegateToFileSystem {
```

Comment:
What is the usage of this class?

Reply:
This is used to identify the `gvfs` prefix for hadoop, any new protocol needs to extend `DelegateToFileSystem`